documents/WCA Bylaws.md

```diff
@@ -20,31 +20,69 @@ The name of this corporation is World Cube Association (“WCA”).
 ### 2.1 Eligibility for Membership
 Application for membership, as defined in Section 5056 of the California Nonprofit Public Benefit Corporation Law, as amended (the “Nonprofit Corporation Law”), shall be open to any person. Membership is granted after completion and receipt of a membership application and annual dues. All memberships shall be granted upon a recommendation of a member and a majority vote of the board, or a majority vote of members at a properly announced meeting. Membership is non-transferrable.
 
-### 2.2	Annual Dues
-The amount required for annual dues shall be $0 each year, unless changed by a majority vote of the members at a properly announced meeting. Continued membership is contingent upon being up-to-date on membership dues.
-
-### 2.3	Rights of Members
-Each member shall be eligible to appoint in writing one voting representative to cast the member’s vote in WCA elections.
-
-### 2.4	Resignation and Termination
+### 2.2 Types of Membership
+The WCA shall offer the following type of memberships to its organization:
+  1. Regional Organizations;
+  2. Associate Regional Organizations;
+  3. Individual voting members;
+  4. Individual non-voting members.
+The Board reserves the right to establish or remove membership categories.  
+  
+### 2.3 Regional Organization Membership
+Regional Organization membership is available for registered entities who act to organize WCA activities within their specified regions. Applications for this type of membership are to be made in writing to the Board of Directors provided the entity meets the following requirements:
+  1. The Regional Organization has a name, address and contact details;
+  2. The Regional Organization has a website and a logo;
+  3. The Regional Organization provides a current copy of it’s Bylaws to the WCA;
+  4. The Regional Organization is registered as a legal entity in the area in which they operate;
```

Comment:
Last part should be "in which it operates", so be consistent with the other usages of "The Regional Organization" (it is always referred to as singular)